docs/battery/usage.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Battery Plus Usage
+sidebar_label: Usage
+hide_title: true
+---
+
+## Using Battery Plus
+
+To use Battery Plus, we provided you with easy to access API. Start by making an instance of `Battery`.
+
+```dart
+final Battery _battery = Battery();
+```
+
+### Access information about the state of device charging
+
+The available states are:
+- **Full**: the battery is at 100%, full of energy.
+- **Charging**: the battery is being charged.
+- **Discharging**: the battery is losing energy.
+
+To get these information in realtime, you can subscribe to the stream from the `Battery` API.
+
+```dart
+// The enum that holds different states of a battery
+BatteryState _batteryState;
+// Create a StreamSubscription that will emit BatteryState values
+StreamSubscription<BatteryState> _batteryStateSubscription;
+
+@override
+void initState() {
+  super.initState();
+
+  // Initialize the subscription by listening to onBatteryStateChanged Stream
+  _batteryStateSubscription =
+      _battery.onBatteryStateChanged.listen((BatteryState state) {
+    
+    // Each time the battery state changes, the local state of the widget is updated
+    setState(() {
+      _batteryState = state;
+    });
+  });
+}
+```
+
+:::note
+Trying to run this code on *Android emulator* may result in a null battery state with a `PlatformException`, but Android emulator provides a way of emulating the different states from its settings. By clicking the menu icon (three vertical dots) then going to battery section.
```

Comment:
> may result in a null battery state

This will be fixed by #134 (initial state for Android) and https://github.com/jpnurmi/plus_plugins/commit/ed049b0536713b09c902615f4ad358be45a99970 (missing "unknown" state).

Reply:
So there will be a forth state "Unknown" ?

Reply:
Yeah, exactly. :+1: I tried both Android & iOS and couldn't reproduce the null state anymore.